gen/src/writer/ucd/mod.rs

```diff
@@ -14,21 +14,25 @@ mod bidi;
 mod case;
 mod category;
 mod core;
+mod ident;
 mod name;
 mod normal;
-mod ident;
+mod segment;
 
 
 use writer::utils::clean_dir;
 
 
 pub fn generate() {
-    age::generate(&clean_dir("unic/ucd/age/tables"));
-    bidi::generate(&clean_dir("unic/ucd/bidi/tables"));
-    case::generate(&clean_dir("unic/ucd/case/tables"));
-    category::generate(&clean_dir("unic/ucd/category/tables"));
-    core::generate(&clean_dir("unic/ucd/core/tables"));
-    name::generate(&clean_dir("unic/ucd/name/tables"));
-    normal::generate(&clean_dir("unic/ucd/normal/tables"));
-    ident::generate(&clean_dir("unic/ucd/ident/tables"));
+    if false {
```

Comment:
Is this left over from debugging?

Reply:
Oops! Thanks for catching it. Will fix and bors.